### DISCUSSION TITLE
[Feature Request] Right click mouse options

### DISCUSSION THREAD

**Discussion by everonegraham** [coddingtonbear/obsidian-web#49] [Ideas]
Hey there,

Thank you for this extension. I just figured out how to get it working too XD.

What would be nice, is to have a right click option in chrome. Like below.

![right click image](https://user-images.githubusercontent.com/28059522/196827332-581f243e-de8f-4fec-b74e-b3e9162553f6.png)

Would reduce the need to keep looking for the extension.

**Comment by mrdarrengriffin** [comment]
Yeah, this would be really nice to have also!
